docs/template_plugin/tests/functional/op_reference/gather_tree.cpp

```diff
@@ -3,91 +3,171 @@
 //
 
 #include <gtest/gtest.h>
+#include <random>
```

Comment:
Why do you need this header?

Reply:
Updated in https://github.com/openvinotoolkit/openvino/pull/9141
Thanks for your comment